### DISCUSSION TITLE
Endpoint cek penggunaan quota pesan pada rentang tanggal tertentu.

### DISCUSSION THREAD

**Discussion by microsauruz** [kirimwa-id/forum#142] [Saran dan Ide]
Halo mas Rio,
ijin menyampaikan usulan lagi, yaitu : fitur untuk mengecek penggunaan quota terhadap hari-hari kemarin mas (search nya range tanggal sekian s.d sekian).
jadi kita bisa tahu tren penggunaan kita secara harian,
bagi yang kreatif, kali2 akan dibikin Dashboard/Grafik naik turunnya penggunaan kirimwa.id

**Comment by rioastamal** [comment]
@microsauruz Thank you usulannya.
